api/config/cdr_config_staging.json

```diff
@@ -104,7 +104,7 @@
     {
       "cdrVersionId": 7,
       "isDefault": true,
-      "name": "Synthetic Dataset in the Controlled Tier",
+      "name": "Synthetic Dataset in the Controlled Tier v4",
```

Comment:
matches the e2e test change in #7190 - the mismatch is what caused this test failure: https://app.circleci.com/pipelines/github/all-of-us/workbench/23399/workflows/048fd592-5617-43d1-bc86-0e36a8d65b6e/jobs/251205

alt: we could revert the e2e test value instead